_posts/2021-03-31-cockpit-241.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Cockpit 241
+author: skobyda
+date: 2021-03-31
+tags: cockpit linux
+slug: cockpit-241
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 241.
+
+## Kdump: Beautification and alignment fixes
+
+This release improves the Kdump page, used for easier kernel debugging. The reworked page now matches the style of the rest of Cockpit and fixes alignment issues.
+
+![Cloud image account initialization](/images/kdump-redesign.png)
+
```

Comment:
You could add a point about how cockpit is improving its certificate installation approach.  Something like


## Start of TLS certificate improvements

Cockpit is starting the process of transitioning away from its (somewhat unique) "merged" form of certificate files, towards the more usual separate files for the certificate and the private key.  This release changes the IPA certificate setup to use the split format, as well as setting the correct permissions at the time the certificate is created.

Watch for more progress on this front in coming releases.